SoftLayer/CLI/core.py

```diff
@@ -69,6 +71,26 @@ def get_command(self, ctx, name):
             return module
 
 
+def get_latest_version():
+    """Gets the latest version of the Softlayer library."""
+    urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+    http = urllib3.PoolManager()
+    try:
+        str_result = http.request('GET', 'https://pypi.python.org/pypi/softlayer/json')
+        dic_result = json.loads(str_result.data.decode('utf-8'))
+        latest = dic_result['info']['version']
+    except Exception:
+        latest = '%(version)s'
+    return latest
+
+
+def get_version_message():
```

Comment:
This will actually call `get_latest_version()` for EVERY command run. So lets do the following:

```python
def get_version_message(ctx, param, value):
    if not value or ctx.resilient_parsing:
        return

    current = SoftLayer.consts.VERSION
    latest = get_latest_version()
    click.secho("Current: {prog} {current}\nLatest:  {prog} v{latest}".format(
        prog=PROG_NAME, current=current, latest=latest))
    ctx.exit()
```